src/modules/draw/draw-directive.js

```diff
@@ -459,10 +496,15 @@ angular.module('anol.draw')
                         modifyControl.disable();
                         modifyControl.interactions = [];
 
+
                         if(angular.isDefined(visibleDewatcher)) {
                             visibleDewatcher();
                         }
 
+                        if (scope.activeLayer && scope.activeLayer.olLayer) {
+                            scope.activeLayer.olLayer.getSource().un('change', setContinueDrawing);
```

Comment:
has to have the same events. see above

Reply:
these should be covered by the `change` event